docs/machine-learning/how-to-guides/consuming-model-ml-net.md

```diff
@@ -1,32 +1,160 @@
 ---
-title: Operationalize a trained machine learning model in apps
+title: "How-To: Save and Load Data Preparation Pipelines and Trained Models"
 description: Discover how to use ML.NET to consume a trained and evaluated machine learning model in applications
-ms.date: 03/05/2019
-ms.custom: mvc,how-to
+ms.date: 04/29/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc, how-to
 #Customer intent: As a developer, I want to use ML.NET to consume my trained and evaluated machine learning model in my applications.
 ---
-# Operationalize a trained machine learning model in apps 
 
-> [!NOTE]
-> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
+# How-To: Save and load trained models
```

Comment:
> How-To: Save and load trained models [](start = 1, length = 37)

as a general feedback, the users don;t necessary need to separate the data prep pipeline, from the model pipeline. The only reason to do that, is to avoid the pain with casting, if they want to access the model parameters, and inspect the model. 

If all they want to do is inference, they are fine to use one pipeline. 

Maybe we can make a note of this, and have one example where the two pipelines are merged.

Reply:
#resolved